### ISSUE TITLE
Problems with suffix

### ISSUE THREAD

**Issue by jelmerkk** [xirixiz/homeassistant-afvalwijzer#83]
I've recently moved to a new adress where I have a suffix "AA" to my house. If I don't fill in the suffix, the mijnafvalwijzer.nl site will tell me they don't recognize my address. With the suffix "AA" it works. But no possible config will work with the integration. The plugin works fine in my setup because if I use my old address it works fine. 

Configuration:
```
platform: afvalwijzer
provider: mijnafvalwijzer
# api_token: None
zipcode: !secret postcode
housenumber: !secret huisnummer
suffix: !secret suffix
# count_today: false
# default_label: Geen
```

Values:
```
postcode: 3511AA
huisnummer: 11
suffix: AA
```

**Comment by jelmerkk** [comment]
BTW I did not publish my actual postal code. But while lookin at the website and trying with some adresses in my old street, there it would also work without the suffix. Maybe that's the culprit...

**Comment by xirixiz** [comment]
Unfortunatly I can only verify what's happening if you can provide me your postal code. A suffix isn't used in the scraper version, but I'm happy to add it if it's really required. Isn't there another housenumber in your street without suffix?

**Comment by xirixiz** [comment]
Just released 4.5.0, this release has the suffix option.

**Comment by jelmerkk** [comment]
That fixed it for me, thanks a lot!
